course_catalog/etl/youtube.py

```diff
@@ -89,60 +89,50 @@ def get_captions_for_video(video):
     Function which fetches and returns xml captions for a video object
 
     Args:
-        video (object) : Video Object
+        video (course_catalog.models.Video)
 
     Returns:
-        a string with the transcript in xml format
+        transcript in xml format (str)
```

Comment:
The format for return types is slightly different: `str: transcript in xml format`